### PR TITLE
cleanup bootstrap

### PR DESCRIPTION
**Pull request by bridiver**
Fixes https://github.com/brave/brave-browser/issues/35184

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used GitHub [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
- Make sure the uplift script works after this merge

### CODE REVIEW

**Review by fmarier — APPROVED** [review]

**Review by goodov — APPROVED** [review]

File: script/bootstrap.py [hunk 1]
```diff
@@ -9,60 +9,28 @@
...
                         help='Prints the output of the subprocesses')
-    parser.add_argument('-y', '--yes', '--assume-yes',
-                        action='store_true',
-                        help='Run non-interactively by assuming "yes" to all '
-                        'prompts.')
-
     return parser.parse_args()
 
 
-def check_root():
```
Comment (bridiver) [inline script/bootstrap.py:52]: oops, this got removed in a rebase. Fixing

File: script/audit_deps.py [hunk 2]
```diff
@@ -14,16 +14,14 @@
...
-import requests
 
 def get_remote_audit_config(
         url = "https://raw.githubusercontent.com/brave/audit-config/main/config.json",
         retry = 3):
-    """Fetch additional audit configuration"""
-    s = requests.Session()
-    s.mount(url, requests.adapters.HTTPAdapter(max_retries=retry))
-    return s.get(url).json()
+    return json.loads(urllib.request.urlopen(url).read().decode("utf-8"))
```
[the remark this answers is not in the document]
Comment (bridiver) [reply script/audit_deps.py:23]: this just changes the library we use to load the file from requests to urllib

File: script/lib/github.py [hunk 3]
```diff
@@ -1,9 +1,11 @@
-#!/usr/bin/env python
+# Copyright (c) 2018 The Brave Authors. All rights reserved.
```
Comment (bridiver) [inline script/lib/github.py:1]: not sure if this is the correct copyright here cc @bbondy

File: script/bump-version.py [hunk 4]
```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```
Comment (bsclifton) [inline script/bump-version.py:1]: Safe to remove - bump logic handled in Jenkins - see https://github.com/brave/devops/blob/c6867ebc0740b6c74cb642084565213cee1b3399/jenkins/jobs/browser/brave-browser-build.yml#L121-L185

File: script/audit_deps.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kdenhartog) [reply script/audit_deps.py:22]: not a concern - the URL is statically defined right above here.

**Review by kdenhartog — APPROVED** [review]
not sure why @fmarier approval isn't satisfying sec-team check, just looked it over again and it LGTM

File: script/lib/connect.py [hunk 5]
```diff
@@ -1,22 +0,0 @@
-import base64
```
Comment (bsclifton) [inline script/lib/connect.py:1]: Also safe to remove; this was only being used by `script/lib/omaha.py` which was removed with https://github.com/brave/brave-core/pull/8543

File: script/lib/github.py (hunk 3 above)
Comment (bsclifton) [reply script/lib/github.py:3]: LGTM - I put a commit through keeping the shebang. License header LGTM

**Review by bsclifton — APPROVED** [review]
++

We'll want to try an uplift after merging this to make sure the GitHub API functionality works as expected

File: script/lib/github.py (hunk 3 above)
Comment (bsclifton) [reply script/lib/github.py:3]: whoops- this caused a linting error. I'll back it out

**Review by goodov — APPROVED** [review]

**Pull request merged**